tests/exporter/collectors/test_network.py

```diff
@@ -45,20 +44,36 @@ def test_get_traffic_in(self):
     def test_get_traffic_out(self):
         """
         @cindyariyo
-        Test if bytes are sent and returned in Mb/s.
+        Test if bytes are sent and returned in Megabits/s.
         """
         network_collector = NetworkCollector()
         traffic_out = network_collector.get_traffic_out()
         self.assertIsInstance(traffic_out, float)
 
+    def test_get_rate_traffic_in(self):
+        """
+        Test if bytes are sent and returned in Megabits/s.
+        """  
+        network_collector = NetworkCollector()
+        rate_traffic_in = network_collector.get_rate_traffic_in()
+        self.assertIsInstance(rate_traffic_in, float)
+
+    def test_get_rate_traffic_out(self):
+        """
+        Test if bytes are sent and returned in Megabits/s.
+        """  
+        network_collector = NetworkCollector()
+        rate_traffic_out = network_collector.get_rate_traffic_out()
+        self.assertIsInstance(rate_traffic_out, float)      
+
     def test__str__(self):
         """
         @cindyariyo
         Test if a string representation of networking metrics is returned.
         """
         network_collector = NetworkCollector()
-        traffic_in = network_collector.get_traffic_in()
-        traffic_out = network_collector.get_traffic_out()
+        traffic_in = network_collector.get_rate_traffic_in()
+        traffic_out = network_collector.get_rate_traffic_out()
         self.assertEqual(network_collector.__str__(), f"Inbound Traffic: {traffic_in} Mb/s, Outbound Traffic: {traffic_out} Mb/s")
```

Comment:
@cindyariyo @ljdzed 
We need to refactor this and make it similar to the RAM one:
![image](https://github.com/max-romagnoli/System-Telemetry-Agent/assets/68606608/8dd19525-bf4e-4ee6-9c49-57ae8f0ac448)

Reply:
I will aim to do this tonight. Will also look into using a class variable to store the previous traffics both for inbound and outbound bytes.

Reply:
@ljdzed @cindyariyo 
Could either of you make the change to the test before this evening?